### ISSUE TITLE
One Finger Click in Center

### ISSUE THREAD

**Issue by DMMeleedy** [rxhanson/Middle-Localization#80]
Is there a way to make this middle click area much narrower? I keep trying to left click and instead I keep getting middle clicks. It's very frustrating as it closes tabs in chrome, and in general, disrupts my use of the magic mouse. If there is a variable that allows one to scale the middle area size that would be ideal, some kind of slider or something.

**Comment by rxhanson** [comment]
This is configurable, but only via Terminal command at this time. I might add it into the UI at some point. See this discussion: https://github.com/rxhanson/Middle-Community/discussions/35

**Comment by DMMeleedy** [comment]
Yes, I was able to set values that made middle click usable for me. I saw that this issue was discussed in 2022. I would vote for that user interface element, so that others can adjust it too.
